### PR TITLE
Fix remote handling of podman images calls

### PR DESCRIPTION
**Pull request by rhatdan**
Enable three more tests
Fix handling of image filters

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@baude @mheon @jwhonce @giuseppe @vrothberg @QiWang19 @TomSweeneyRedHat PTAL

File: test/e2e/images_test.go [hunk 1]
```diff
@@ -152,9 +152,7 @@ var _ = Describe("Podman images", func() {
 	})
 
 	It("podman images filter reference", func() {
-		if podmanTest.RemoteTest {
-			Skip("Does not work on remote client")
-		}
+		SkipIfRemote()
```
Comment (QiWang19) [inline test/e2e/images_test.go:155]: Does this enable the test?

Comment (baude) [reply test/e2e/images_test.go:155]: it seems like he is replacing a piece of code with a convenience function.

**Comment by baude** [comment]
lgtm

File: test/e2e/images_test.go (hunk 1 above)
Comment (rhatdan) [reply test/e2e/images_test.go:155]: Yes just cleaning up.
BTW These can get enabled for remote, as soon as podman-remote build works.

File: test/e2e/images_test.go [hunk 2]
```diff
@@ -180,9 +178,7 @@ var _ = Describe("Podman images", func() {
 	})
 
 	It("podman images filter before image", func() {
-		if podmanTest.RemoteTest {
-			Skip("Does not work on remote client")
-		}
+		SkipIfRemote()
```
Comment (mheon) [inline test/e2e/images_test.go:181]: Why are all of these SkipIfRemote? A lot of them seem like they ought to work remote

Comment (rhatdan) [reply test/e2e/images_test.go:181]: podman build does not work, so these will not work until that is fixed.  These were blocked in podman 1.9 as well.

Comment (mheon) [reply test/e2e/images_test.go:181]: Ahh, ack.

**Pull request merged**